README.md

```diff
@@ -411,19 +411,17 @@ Supported compilers include:
 
 #### Linux*
 
-Operating System | CPU Host/Target | Integrated Graphics from Intel (Intel GPU) |  NVIDIA GPU
+Operating System | x86 CPU | Intel GPU |  NVIDIA GPU
 :--- | :--- | :--- | :---
-Ubuntu                            | 18.04.3, 19.04 | 18.04.3, 19.10  | 18.04.3, 20.04
-SUSE Linux Enterprise Server*     | 15             | *Not supported* | *Not supported*
-Red Hat Enterprise Linux* (RHEL*) | 8              | *Not supported* | *Not supported*
-Linux* kernel                     | *N/A*          | 4.11 or higher | *N/A*
+Ubuntu                            | *Not supported* | 22.04 LTS  | 22.04 LTS
```

Comment:
Why is `x86 CPU` no longer supported? What does this mean for the MKLCPU backend?

Reply:
I think the table format is causing confusion. I reorganized it into </br>
![Screenshot 2024-03-25 134556](https://github.com/oneapi-src/oneMKL/assets/91495331/c2be4d7b-cd83-4ba3-aba8-f9d2c4988bc3)

Does it look better and make sense now?

Reply:
Unfortunately, I still find this confusing:
* Should I expect the MKLCPU backend to work on Ubuntu 22 LTS?
  * I think yes, but the above table suggests that I need RHEL9 for that.

It also occurs to me that there is no mention of AMD support.

Reply:
We can only claim that an OS is supported if it is in our CI. For an OS that is not listed, it could still work, but there is no guarantee.

Reply:
OK. Its a surprise to me MKLCPU isn't tested on Ubuntu 22, but since it isn't the table makes sense. Thank for clarifying that!